packages/react-app-revamp/hooks/useContest/index.ts

```diff
@@ -247,7 +268,7 @@ export function useContest() {
       if (
         process.env.NEXT_PUBLIC_SUPABASE_URL !== "" &&
         process.env.NEXT_PUBLIC_SUPABASE_URL &&
-        process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY !== "" &&
+        process.env.NEXT_getCurrentSubmissionTokenVotesPUBLIC_SUPABASE_ANON_KEY !== "" &&
```

Comment:
What is this env var?

Reply:
oops, moved around some code around this file and this happened :eyes:  reversing this !

Reply:
Dope, thank you!